packages/airnode-deployer/terraform/airnode/aws/variables.tf

```diff
@@ -38,3 +38,8 @@ variable "api_key" {
   type        = string
   default     = null
 }
+
+variable "api_max_concurrency" {
+  description = "Maximum amount of concurrent executions for Airnode Test Gateway Lambda"
+  default = -1
```

Comment:
Shouldn't this be 0?

Reply:
No, in AWS 0 means disabled (no instance can be run) and -1 means no reservations. In GCP you can't disable a function with this option, so 0 means no reservations. :man_shrugging: